### DISCUSSION TITLE
How to use Decision Node Functionality?

### DISCUSSION THREAD

**Discussion by gomugomu0034** [gorules/zen-go#16] [Q&A]
I have been trying to explore the decision node functionality with a simple flow but getting an error. 
Details - 
Node error
Loader failed internally on key test/decision_new.json: missing field `nodes` at line 1 column 2.

Main policy - 
[decision.json](https://github.com/gorules/zen-go/files/15184945/decision.json)
Inherited policy - 
[decision_new.json](https://github.com/gorules/zen-go/files/15184842/decision_new.json)

Can someone review if i am doing things correctly? I am unable to figure out the solution for this error.

Thanks in Advance

**Comment by gomugomu0034** [comment]
figured out. I didn't publish the inherited policy before testing. It worked once i published it

**Comment by ivanmiletic** [comment]
Hi, glad you managed to solve it. Ill close this issue
